Dockerfile.dev

```diff
@@ -0,0 +1,35 @@
+FROM golang:1.23.2-alpine
+
+WORKDIR /app
+
+# Install required system dependencies
+RUN apk add --no-cache \
+    gcc \
+    musl-dev \
+    git \
+    nodejs \
+    npm \
+    curl \
+    make
+
+# Install Air for hot reloading
+RUN curl -sSfL https://raw.githubusercontent.com/cosmtrek/air/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.61.5
+
+# Install TailwindCSS
+RUN npm install -g tailwindcss@3.4.13
```

Comment:
Use installation instructions from the prod file. To install the standalone api without the need for nodejs